docs/particles/particle_class.rst

```diff
@@ -259,6 +259,18 @@ and/or |ParticleList| objects together.
 >>> helium_ions + cp + proton
 ParticleList(['He-4 0+', 'He-4 1+', 'He-4 2+', 'Fe 9.5+', 'p+'])
 
+As with an individual |Particle| and |CustomParticle|, we can check whether
+all the particles in a list fall within a category using |is_category|.
+
+>>> helium_ions.is_category("ion")
+False
+
+We may also check each particle in the list individually by setting
+the keyword ``particlewise`` to `True`.
+
+>>> helium_ions.is_category("ion", particlewise=True)
+[False, True, True]
+
```

Comment:
👍🏻 Thank you for updating this here! I often forget to update the narrative docs until I realize it about six months later... 😅

Reply:
I only caught this while trying to track down why the docs were failing to build